src/consumeAPI.js

```diff
@@ -0,0 +1,48 @@
+import { populateScores } from './DynamicHTML.js';
+
+export const createGame = async () => {
+  let gameID;
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'StreetFighter2',
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json())
+    .then((name) => {
+      gameID = name.result.split(' ')[name.result.split(' ').indexOf('ID:') + 1];
+      alert(gameID);
```

Comment:
- [ ] Please do not use this alert() method as it stops code execution and displays differently on different browsers. Kindly remove it 👍 .